prow/plugins/cat/cat_test.go

```diff
@@ -121,6 +123,15 @@ func TestUrl(t *testing.T) {
 			movie:    true,
 			require:  []string{"category=this", "api_key=that", "&", "mime_types=gif"},
 		},
+		{
+			name:     "grumpy cat",
```

Comment:
Please remove this test case, it doesn't actually check the grumpy cat functionality.